bases/shared/entrypoint.sh

```diff
@@ -471,6 +476,8 @@ fork_setup() {
     persistent_peers="persistent_peers = \"0663e8221928c923d516ea1e8972927f54da9edb@$FORK1_IP:26656,e234dc7fffdea593c5338a9dd8b5c22ba00731eb@$FORK2_IP:26656\""
     sed -i "/^persistent_peers =/s/.*/$persistent_peers/" $AGORIC_HOME/config/config.toml
 
+    sed -i "/^snapshot-interval = 0/snapshot-interval = 1000/" $AGORIC_HOME/config/app.toml
```

Comment:
Do we actually need this? I assume it's to allow higher fidelity testing with load introduced by ongoing snapshotting, but want to confirm.